app/person.py

```diff
@@ -0,0 +1,56 @@
+
+""" Amity models"""
+
+
+class Person(object):
+    """docstring for Person."""
```

Comment:
This is not how you write a function doc string. Please put a better description than what is there.